docs/pages/docs/getting-started/app-router-client-components.mdx

```diff
@@ -77,7 +77,7 @@ export function generateStaticParams() {
 export default async function LocaleLayout({children, params: {locale}}) {
   let messages;
   try {
-    messages = (await import(`../../messages/${locale}.json`)).default;
```

Comment:
This matches the directory structure outlined at the top here, doesn't it?

https://next-intl-docs.vercel.app/docs/getting-started/app-router-client-components#getting-started

Reply:
Sorry, it was my mistake.

Reply:
No worries!

FWIW the default for the `src` directory config in the Next.js CLI was switched to "Yes" now:

<img width="493" alt="Screenshot 2023-08-02 at 09 48 36" src="https://github.com/amannn/next-intl/assets/4038316/59df04c2-318f-496a-8bd0-2c4b82f0f494">

So I think it would make sense to generally move to that.